staging/src/k8s.io/apiserver/pkg/storage/cacher/watch_cache.go

```diff
@@ -629,17 +629,22 @@ const (
 	// maxWatchChanSizeWithoutIndex is the max size of the channel
 	// used by the watch not using the index.
 	// TODO(wojtek-t): Figure out if the value shouldn't be higher.
-	maxWatchChanSizeWithoutIndex = 100
+	maxWatchChanSizeWithoutIndex = 1000
```

Comment:
I'm against doing this.

The pattern that you described in the parent bug (watching frequently changing resources from every node) is an anti-pattern of doing things. By allowing larger buffers we effectively increase memory usage of apiserver and I don't want to do that without a better reason.

Reply:
Is it better to add a flag here for users to tune?

Reply:
That I would be fine with.